workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -30,23 +33,33 @@ private static void Generate()
 
                 DownloadCoreSdk.Download();
 
-                CopySchema(SchemaRoot);
+                CopySchema(SchemaRootDir);
 
                 var projectPath = Path.GetFullPath(Path.Combine(Common.GetThisPackagePath(),
-                    ".CodeGenerator/GdkCodeGenerator/GdkCodeGenerator.csproj"));
-                var schemaCompilerPath = Path.GetFullPath(Path.Combine(Application.dataPath,
-                    $"../build/CoreSdk/{Common.CoreSdkVersion}/schema_compiler/schema_compiler"));
+                    CsProjectFile));
 
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                var schemaCompilerPath =
+                    Path.GetFullPath(Path.Combine(Application.dataPath, schemaCompilerRelativePath));
+
+                switch (Application.platform)
                 {
-                    schemaCompilerPath = Path.ChangeExtension(schemaCompilerPath, ".exe");
+                    case RuntimePlatform.WindowsEditor:
+                        schemaCompilerPath = Path.ChangeExtension(schemaCompilerPath, ".exe");
+                        break;
+                    case RuntimePlatform.LinuxEditor:
+                    case RuntimePlatform.OSXEditor:
+                        // Ensure the schema compiler is executable.
+                        var _ = Common.RunProcess("chmod", "+x", schemaCompilerPath);
```

Comment:
Can't we just call `Common.RunProcess(...)` if  we don't care about the return variable, can just omit the binding entirely right?

Reply:
I thought it nicer to acknowledge we're actively ignoring the result, rather than forgetting to deal with it.

Reply:
That makes sense 👍